internal/storage/storage.go

```diff
@@ -134,6 +134,12 @@ func (prs PackageVersions) Strings() []string {
 
 // CloneRepository function clones the repository and changes branch to stage.
```

Comment:
Now that this function depends on a fork, maybe update the godoc comment here to say that?

Reply:
Fixed